tests/appsec/waf/test_telemetry.py

```diff
@@ -0,0 +1,88 @@
+from utils import interfaces, released, rfc, weblog, coverage, scenario, context
+from utils.tools import logger
+
+TELEMETRY_REQUEST_TYPE_GENERATE_METRICS = "generate-metrics"
+
+
+def _validate_headers(headers, language):
```

Comment:
`language` is always equals to `context.library`

```suggestion
def _validate_metrics(payload):
```